### PR TITLE
Add powertrain switch TLE9104

### PR DESCRIPTION
**Pull request by benediktibk**
Implement a driver for the powertrain switch TLE9104 from Infineon.

### CODE REVIEW

File: drivers/gpio/gpio_tle9104.c
Comment (ifyall) [inline drivers/gpio/gpio_tle9104.c:22]: It would be best to start this file with a comment documenting where the various values and definitions were sourced from. This applies to constants like these as well as the register definitions. A link to a datasheet or other source would be acceptable.

**Review by ifyall — DISMISSED** [review]

File: drivers/gpio/gpio_tle9104.c
Comment (benediktibk) [reply drivers/gpio/gpio_tle9104.c:27]: done

**Comment by benediktibk** [comment]
Rebased to resolve merge conflicts.

File: drivers/gpio/Kconfig.tle9104
Comment (henrikbrixandersen) [inline drivers/gpio/Kconfig.tle9104:10]:
```suggestion
	select SPI
```

File: drivers/gpio/Kconfig.tle9104
Comment (henrikbrixandersen) [inline drivers/gpio/Kconfig.tle9104:22]:
```suggestion
config GPIO_TLE9104_INIT_PRIORITY
	int "Init priority"
	depends on GPIO_TLE9104
	default 75
	help
	  Device driver initialization priority.
```

File: drivers/gpio/gpio_tle9104.c
Comment (henrikbrixandersen) [inline drivers/gpio/gpio_tle9104.c:69]:
```suggestion
	struct gpio_driver_data common;
```

File: drivers/gpio/gpio_tle9104.c
Comment (henrikbrixandersen) [inline drivers/gpio/gpio_tle9104.c:362]: Please add `ARG_UNUSED()` macro calls for all unused arguments.

File: dts/bindings/gpio/infineon,tle9104.yaml
Comment (henrikbrixandersen) [inline dts/bindings/gpio/infineon,tle9104.yaml:46]: I am not sure what the purpose of these properties is?

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]

File: dts/bindings/gpio/infineon,tle9104.yaml
Comment (benediktibk) [reply dts/bindings/gpio/infineon,tle9104.yaml:46]: There are two possibilities to control the outputs, either via SPI commands or via GPIOs. Configuring one of these will ensure that the outputs are controlled via them, reducing the number of necessary SPI commands. But is not required to use them, if they are not set the outputs will be controlled via SPI commands.

File: drivers/gpio/Kconfig.tle9104
Comment (benediktibk) [reply drivers/gpio/Kconfig.tle9104]: done

File: drivers/gpio/gpio_tle9104.c
Comment (benediktibk) [reply drivers/gpio/gpio_tle9104.c]: done

File: drivers/gpio/Kconfig.tle9104
Comment (benediktibk) [reply drivers/gpio/Kconfig.tle9104:10]: No can do, this would result in a dependency loop:

    Parsing /home/bschmidt/Development/zephyr_playground/zephyr/Kconfig
    /home/bschmidt/Development/zephyr_playground/zephyr/scripts/kconfig/kconfig.py: Dependency loop
    ===============
    
    SPI (defined at boards/shields/x_nucleo_idb05a1/Kconfig.defconfig:8, soc/arm/nxp_kinetis/k6x/Kconfig.defconfig.mk64f12:18, soc/arm/nxp_kinetis/kwx/Kconfig.defconfig.mkw2xd512:19, drivers/spi/Kconfig:9), with definition...
    
    config SPI
        bool
        default y
        depends on BT && SHIELD_X_NUCLEO_IDB05A1
    
    config SPI
        bool
        default n
        depends on SOC_MK64F12 && SOC_SERIES_KINETIS_K6X
    
    config SPI
        bool
        default y
        depends on (SOC_MKW22D5 || SOC_MKW24D5) && SOC_SERIES_KINETIS_KWX
    
    menuconfig SPI
        bool "Serial Peripheral Interface (SPI) bus drivers"
        help
            Enable support for the SPI hardware bus.
    
    (select-related dependencies: (ADC_LMP90XXX && (DT_HAS_TI_LMP90077_ENABLED || DT_HAS_TI_LMP90078_ENABLED || DT_HAS_TI_LMP90079_ENABLED || DT_HAS_TI_LMP90080_ENABLED || DT_HAS_TI_LMP90097_ENABLED || DT_HAS_TI_LMP90098_ENABLED || DT_HAS_TI_LMP90099_ENABLED || DT_HAS_TI_LMP90100_ENABLED) && ADC) || (ADC_MCP320X && (DT_HAS_MICROCHIP_MCP3204_ENABLED || DT_HAS_MICROCHIP_MCP3208_ENABLED) && ADC) || (ADC_ADS7052 && DT_HAS_TI_ADS7052_ENABLED && ADC) || (ADC_ADS114S0X && DT_HAS_TI_ADS114S08_ENABLED && ADC) || (ADC_MAX1125X && (DT_HAS_MAXIM_MAX11254_ENABLED || DT_HAS_MAXIM_MAX11253_ENABLED) && ADC) || (AUXDISPLAY_PT6314 && DT_HAS_PTC_PT6314_ENABLED && AUXDISPLAY) || (CAN_MCP2515 && DT_HAS_MICROCHIP_MCP2515_ENABLED && CAN) || (DAC_DACX0508 && (DT_HAS_TI_DAC60508_ENABLED || DT_HAS_TI_DAC70508_ENABLED || DT_HAS_TI_DAC80508_ENABLED) && DAC) || (DAC_LTC166X && (DT_HAS_LLTC_LTC1660_ENABLED || DT_HAS_LLTC_LTC1665_ENABLED) && DAC) || (ILI9340 && DT_HAS_ILITEK_ILI9340_ENABLED && DISPLAY) || (ILI9341 && DT_HAS_ILITEK_ILI9341_ENABLED && DISPLAY) || (ILI9342C && DT_HAS_ILITEK_ILI9342C_ENABLED && DISPLAY) || (ILI9488 && DT_HAS_ILITEK_ILI9488_ENABLED && DISPLAY) || (SSD16XX && (DT_HAS_SOLOMON_SSD1608_ENABLED || DT_HAS_SOLOMON_SSD1673_ENABLED || DT_HAS_SOLOMON_SSD1675A_ENABLED || DT_HAS_SOLOMON_SSD1680_ENABLED || DT_HAS_SOLOMON_SSD1681_ENABLED) && DISPLAY) || (ST7735R && DT_HAS_SITRONIX_ST7735R_ENABLED && DISPLAY) || (ST7789V && DT_HAS_SITRONIX_ST7789V_ENABLED && DISPLAY) || (UC81XX && (DT_HAS_ULTRACHIP_UC8176_ENABLED || DT_HAS_ULTRACHIP_UC8179_ENABLED) && DISPLAY) || (LS0XX && DT_HAS_SHARP_LS0XX_ENABLED && DISPLAY) || (MAX7219 && DT_HAS_MAXIM_MAX7219_ENABLED && DISPLAY) || (EEPROM_AT25 && DT_HAS_ATMEL_AT25_ENABLED && EEPROM) || (ETH_ENC28J60 && DT_HAS_MICROCHIP_ENC28J60_ENABLED && ETH_DRIVER) || (ETH_ENC424J600 && DT_HAS_MICROCHIP_ENC424J600_ENABLED && ETH_DRIVER) || (ETH_W5500 && DT_HAS_WIZNET_W5500_ENABLED && ETH_DRIVER) || (ETH_ADIN2111 && (DT_HAS_ADI_ADIN2111_ENABLED || DT_HAS_ADI_ADIN1110_ENABLED) && ETH_DRIVER) || (SPI_FLASH_AT45 && DT_HAS_ATMEL_AT45_ENABLED && FLASH) || (SPI_NOR && DT_HAS_JEDEC_SPI_NOR_ENABLED && FLASH) || (GPIO_TLE9104 && DT_HAS_INFINEON_TLE9104_ENABLED && GPIO) || (IEEE802154_MCR20A && DT_HAS_NXP_MCR20A_ENABLED && IEEE802154) || (IEEE802154_RF2XX && DT_HAS_ATMEL_RF2XX_ENABLED && IEEE802154) || (IEEE802154_DW1000 && DT_HAS_DECAWAVE_DW1000_ENABLED && IEEE802154) || (INPUT_XPT2046 && DT_HAS_XPTEK_XPT2046_ENABLED && INPUT) || (FT800 && DT_HAS_FTDI_FT800_ENABLED) || (SPI_SDHC && DT_HAS_ZEPHYR_SDHC_SPI_SLOT_ENABLED && SDHC) || (ADT7310 && DT_HAS_ADI_ADT7310_ENABLED && SENSOR) || (ADXL362 && DT_HAS_ADI_ADXL362_ENABLED && SENSOR) || (BMI08X && (n || n) && (DT_HAS_BOSCH_BMI08X_ACCEL_ENABLED || DT_HAS_BOSCH_BMI08X_GYRO_ENABLED) && SENSOR) || (I3G4250D && DT_HAS_ST_I3G4250D_ENABLED && SENSOR) || (ICM42605 && DT_HAS_INVENSENSE_ICM42605_ENABLED && SENSOR) || (ICM42670 && DT_HAS_INVENSENSE_ICM42670_ENABLED && SENSOR) || (ICM42688 && DT_HAS_INVENSENSE_ICM42688_ENABLED && SENSOR) || (IIS3DHHC && DT_HAS_ST_IIS3DHHC_ENABLED && SENSOR) || (MAX31855 && DT_HAS_MAXIM_MAX31855_ENABLED && SENSOR) || (MAX31865 && DT_HAS_MAXIM_MAX31865_ENABLED && SENSOR) || (MAX6675 && DT_HAS_MAXIM_MAX6675_ENABLED && SENSOR) || (UHC_MAX3421E && DT_HAS_MAXIM_MAX3421E_SPI_ENABLED && UHC_DRIVER) || (WDT_NXP_FS26 && DT_HAS_NXP_FS26_WDOG_ENABLED && WATCHDOG) || (WIFI_WINC1500 && DT_HAS_ATMEL_WINC1500_ENABLED && WIFI) || (WIFI_ESWIFI_BUS_SPI && <choice WIFI_ESWIFI_BUS>))
    
    ...depends on GPIO (defined at soc/arm/microchip_mec/mec1501/Kconfig.defconfig.mec1501hsz:11, soc/arm/microchip_mec/mec172x/Kconfig.defconfig.mec172xnlj:11, soc/arm/microchip_mec/mec172x/Kconfig.defconfig.mec172xnsz:11, soc/arm/nordic_nrf/Kconfig.defconfig:39, soc/arm/nxp_imx/mcimx7_m4/Kconfig.defconfig.mcimx7_m4:14, soc/arm/nxp_imx/mimx8ml8_m7/Kconfig.defconfig.mimx8ml8_m7:16, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1010:18, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1015:17, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1021:17, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1024:17, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1042:12, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1052:14, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1062:14, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1064:14, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1166_cm4:14, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1166_cm7:14, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1176_cm4:14, soc/arm/nxp_imx/rt/Kconfig.defconfig.mimxrt1176_cm7:14, soc/arm/nxp_kinetis/k2x/Kconfig.defconfig.mk22f12:11, soc/arm/nxp_kinetis/k6x/Kconfig.defconfig.mk64f12:15, soc/arm/nxp_kinetis/k6x/Kconfig.defconfig.mk66f18:15, soc/arm/nxp_kinetis/k8x/Kconfig.defconfig.series:22, soc/arm/nxp_kinetis/ke1xf/Kconfig.defconfig.series:36, soc/arm/nxp_kinetis/kv5x/Kconfig.defconfig.series:15, soc/arm/nxp_lpc/lpc54xxx/Kconfig.defconfig.lpc54114_m0:11, soc/arm/nxp_lpc/lpc55xxx/Kconfig.defconfig.lpc55S69_cpu1:11, drivers/gpio/Kconfig:6), with definition...
    
    config GPIO
        bool
        default y
        depends on SOC_MEC1501_HSZ && SOC_SERIES_MEC1501X
    
    config GPIO
        bool
        default y
        depends on SOC_MEC172X_NLJ && SOC_SERIES_MEC172X
    
    config GPIO
        bool
        default y
        depends on SOC_MEC172X_NSZ && SOC_SERIES_MEC172X
    
    config GPIO
        bool
        default y
        depends on SPI && SOC_FAMILY_NRF
    
    config GPIO
        bool
        default y
        depends on SOC_MCIMX7_M4 && SOC_SERIES_IMX7_M4
    
    config GPIO
        bool
        default y
        depends on SOC_MIMX8ML8 && SOC_SERIES_IMX8ML_M7
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1011 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1015 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1021 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1024 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1042 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1052 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1062 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1064 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1166_CM4 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1166_CM7 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1176_CM4 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MIMXRT1176_CM7 && SOC_SERIES_IMX_RT
    
    config GPIO
        bool
        default y
        depends on SOC_MK22F51212 && SOC_SERIES_KINETIS_K2X
    
    config GPIO
        bool
        default y
        depends on SOC_MK64F12 && SOC_SERIES_KINETIS_K6X
    
    config GPIO
        bool
        default y
        depends on SOC_MK66F18 && SOC_SERIES_KINETIS_K6X
    
    config GPIO
        bool
        default y
        depends on SOC_SERIES_KINETIS_K8X
    
    config GPIO
        bool
        default y
        depends on SOC_SERIES_KINETIS_KE1XF
    
    config GPIO
        bool
        default y
        depends on SOC_SERIES_KINETIS_KV5X
    
    config GPIO
        bool
        default n
        depends on SOC_LPC54114_M0 && SOC_SERIES_LPC54XXX
    
    config GPIO
        bool
        default y
        depends on SOC_LPC55S69_CPU1 && SOC_SERIES_LPC55XXX
    
    menuconfig GPIO
        bool "General-Purpose Input/Output (GPIO) drivers"
        help
            Include GPIO drivers in system config
    
    (select-related dependencies: (AUXDISPLAY_ITRON && DT_HAS_NORITAKE_ITRON_ENABLED && AUXDISPLAY) || (IEEE802154_RF2XX && DT_HAS_ATMEL_RF2XX_ENABLED && IEEE802154) || (REGULATOR_GPIO && DT_HAS_REGULATOR_GPIO_ENABLED && REGULATOR) || (SPI_SAM && DT_HAS_ATMEL_SAM_SPI_ENABLED && SPI) || (SPI_XMC4XXX && DT_HAS_INFINEON_XMC4XXX_SPI_ENABLED && SPI) || (WDT_NXP_FS26 && DT_HAS_NXP_FS26_WDOG_ENABLED && WATCHDOG) || (WIFI_ESWIFI && (DT_HAS_INVENTEK_ESWIFI_ENABLED || DT_HAS_INVENTEK_ESWIFI_UART_ENABLED) && WIFI))
    
    (imply-related dependencies: (AUXDISPLAY_ITRON && DT_HAS_NORITAKE_ITRON_ENABLED && AUXDISPLAY) || (IEEE802154_RF2XX && DT_HAS_ATMEL_RF2XX_ENABLED && IEEE802154) || (REGULATOR_GPIO && DT_HAS_REGULATOR_GPIO_ENABLED && REGULATOR) || (SPI_SAM && DT_HAS_ATMEL_SAM_SPI_ENABLED && SPI) || (SPI_XMC4XXX && DT_HAS_INFINEON_XMC4XXX_SPI_ENABLED && SPI) || (WDT_NXP_FS26 && DT_HAS_NXP_FS26_WDOG_ENABLED && WATCHDOG) || (WIFI_ESWIFI && (DT_HAS_INVENTEK_ESWIFI_ENABLED || DT_HAS_INVENTEK_ESWIFI_UART_ENABLED) && WIFI))
    
    ...depends on SPI_SAM (defined at drivers/spi/Kconfig.sam:7), with definition...
    
    config SPI_SAM
        bool "Atmel SAM series SPI driver"
        default y
        select GPIO
        depends on DT_HAS_ATMEL_SAM_SPI_ENABLED && SPI
        help
            Enable support for the SAM SPI driver.
    
    ...depends again on SPI (defined at boards/shields/x_nucleo_idb05a1/Kconfig.defconfig:8, soc/arm/nxp_kinetis/k6x/Kconfig.defconfig.mk64f12:18, soc/arm/nxp_kinetis/kwx/Kconfig.defconfig.mkw2xd512:19, drivers/spi/Kconfig:9)

Also, it looks like other drivers (BD8LB600FS, CY8C95XX, FXL6408, MCP23S17, ...) seem to handle this the same with depends on instead of select.

File: dts/bindings/gpio/infineon,tle9104.yaml
Comment (henrikbrixandersen) [reply dts/bindings/gpio/infineon,tle9104.yaml:46]: Shouldn't this just be one property, then? E.g. `in-gpios`?

**Review by henrikbrixandersen — CHANGES_REQUESTED** [review]

File: dts/bindings/gpio/infineon,tle9104.yaml
Comment (benediktibk) [reply dts/bindings/gpio/infineon,tle9104.yaml:46]: I thought about this option as well, but then the user would have to specify all the gpios or none. And it is actually possible to control the outputs only partially via dedicated GPIOs and partially via SPI. I could combine something like in-gpios with a mask property, but I am not certain if this would make the proper usage easier to understand.

Comment (henrikbrixandersen) [reply dts/bindings/gpio/infineon,tle9104.yaml:46]: Thank you, let's leave them as separate properties, then.

**Review by henrikbrixandersen — DISMISSED** [review]
LGTM

**Comment by benediktibk** [comment]
Rebased to resolve merge conflicts.

**Comment by benediktibk** [comment]
@ifyall Could you please take another look?

**Review by henrikbrixandersen — DISMISSED** [review]

**Comment by benediktibk** [comment]
Rebased to resolve merge conflicts

**Comment by benediktibk** [comment]
Fixed build error on intel_adl_crb

**Review by henrikbrixandersen — DISMISSED** [review]

**Comment by benediktibk** [comment]
rebased to resolve merge conflicts

**Review by henrikbrixandersen — DISMISSED** [review]

File: dts/bindings/gpio/infineon,tle9104.yaml
Comment (fabiobaltieri) [inline dts/bindings/gpio/infineon,tle9104.yaml:21]: nit: GPIOs

File: drivers/gpio/gpio_tle9104.c
Comment (fabiobaltieri) [inline drivers/gpio/gpio_tle9104.c:31]: nit: time defines suffixes are normally just the constant, like `TIMEOUT_MS` not `TIMEOUT_IN_MS`, there's few cases with the `IN` as well but I suggest sticking to the shorter version.... it's... shorter :-)

File: drivers/gpio/gpio_tle9104.c
Comment (fabiobaltieri) [inline drivers/gpio/gpio_tle9104.c:168]: add a define for `BIT(13)`

File: drivers/gpio/gpio_tle9104.c
Comment (fabiobaltieri) [inline drivers/gpio/gpio_tle9104.c:256]: add a define for "7" (shouldn't it be 4? like `TLE9104_GPIO_COUNT`)

File: drivers/gpio/gpio_tle9104.c
Comment (fabiobaltieri) [inline drivers/gpio/gpio_tle9104.c:311]: nit: should the `ARG_UNUSED` be first?

File: drivers/gpio/gpio_tle9104.c
Comment (fabiobaltieri) [inline drivers/gpio/gpio_tle9104.c:518]: the curly brace should be inline for these (I know clang-format is messing these up, sorry, no idea how to fix that)

File: drivers/gpio/gpio_tle9104.c
Comment (fabiobaltieri) [inline drivers/gpio/gpio_tle9104.c:537]: no need to initialize this, static data is zero initialized automatically

File: drivers/gpio/gpio_tle9104.c
Comment (alexanderwachter) [inline drivers/gpio/gpio_tle9104.c:96]: Hint:
You could use popcount here

File: drivers/gpio/gpio_tle9104.c
Comment (benediktibk) [reply drivers/gpio/gpio_tle9104.c:514]: Do you mean it breaks https://gitlab.com/MISRA/MISRA-C/MISRA-C-2012/Example-Suite/-/blob/master/R_09_02.c of https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html?

And what would be the correct way, maybe this?

		.common = {                                                                         \
				.port_pin_mask = GPIO_PORT_PIN_MASK_FROM_DT_INST(inst),            \
			},                                                                         \

File: dts/bindings/gpio/infineon,tle9104.yaml
Comment (benediktibk) [reply dts/bindings/gpio/infineon,tle9104.yaml]: done

File: drivers/gpio/gpio_tle9104.c
Comment (benediktibk) [reply drivers/gpio/gpio_tle9104.c]: done

File: drivers/gpio/gpio_tle9104.c
Comment (benediktibk) [reply drivers/gpio/gpio_tle9104.c]: done

File: drivers/gpio/gpio_tle9104.c
Comment (benediktibk) [reply drivers/gpio/gpio_tle9104.c]: nice catch, done

File: drivers/gpio/gpio_tle9104.c
Comment (benediktibk) [reply drivers/gpio/gpio_tle9104.c]: done

File: drivers/gpio/gpio_tle9104.c
Comment (benediktibk) [reply drivers/gpio/gpio_tle9104.c]: done

File: drivers/gpio/gpio_tle9104.c
Comment (benediktibk) [reply drivers/gpio/gpio_tle9104.c]: Learned something new today, thanks.
done

Comment (fabiobaltieri) [reply drivers/gpio/gpio_tle9104.c]: damn, cool!

**Review by fabiobaltieri — DISMISSED** [review]

**Comment by benediktibk** [comment]
rebased to resolve merge conflicts

**Review by fabiobaltieri — DISMISSED** [review]

**Review by henrikbrixandersen — DISMISSED** [review]

**Review by alexanderwachter — DISMISSED** [review]

**Comment by benediktibk** [comment]
rebased to resolve merge conflicts

**Review by fabiobaltieri — DISMISSED** [review]

**Review by henrikbrixandersen — DISMISSED** [review]

**Comment by henrikbrixandersen** [comment]
@ifyall You change request was addressed, as far as I can tell. Please either revisit this or dismiss your review, as it is blocking this PR from being merged.

**Review by fabiobaltieri — DISMISSED** [review]

**Comment by benediktibk** [comment]
rebased to resolve merge conflicts

**Comment by henrikbrixandersen** [comment]
You should drop the change to CODEOWNERS.

**Comment by benediktibk** [comment]
> You should drop the change to CODEOWNERS.

I actually thought about this, but with the last big change my other entries there stayed. Therefore I thought I should addding them?

I personally do not have a hard opinion on this topic, I just got the feedback on a previous review that it is required to add the entries in CODEOWNERS. But I'll happily drop this commit.

**Comment by henrikbrixandersen** [comment]
> I actually thought about this, but with the last big change my other entries there stayed. Therefore I thought I should keep adding them?
> 
> I personally do not have a hard opinion on this topic, I just got the feedback on a previous review that it is required to add the entries in CODEOWNERS. But I'll happily drop this commit.

It is not required, the file will soon be deprecated. Please see the new instructions at the very top of https://github.com/zephyrproject-rtos/zephyr/blob/main/CODEOWNERS

**Review by fabiobaltieri — APPROVED** [review]

**Review by henrikbrixandersen — APPROVED** [review]

**Comment by benediktibk** [comment]
> > I actually thought about this, but with the last big change my other entries there stayed. Therefore I thought I should keep adding them?
> > I personally do not have a hard opinion on this topic, I just got the feedback on a previous review that it is required to add the entries in CODEOWNERS. But I'll happily drop this commit.
> 
> It is not required, the file will soon be deprecated. Please see the new instructions at the very top of https://github.com/zephyrproject-rtos/zephyr/blob/main/CODEOWNERS

Removed the last commit which added the entry to the CODEOWNERS.

**Pull request merged**
